components/dashboard/TrustScore.tsx

```
@@ -0,0 +1,112 @@
+            {/* Percentage number */}
+            <span className="font-utsaha text-2xl font-medium text-[#0553fd] md:text-3xl">
+              {clampedScore}%
+            </span>
+
+            {/* Arc meter */}
+            <div className="h-[50px] w-[90px] md:h-[60px] md:w-[110px]">
+              <svg
+                viewBox="0 0 100 55"
+                className="h-full w-full"
+                style={{ overflow: "visible" }}
+              >
```

Biome (2.4.6)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)